src/server/node_func.c

```diff
@@ -2525,7 +2522,7 @@ remove_node_topology(char *node_name)
 			PBS_EVENTCLASS_SERVER,
 			LOG_DEBUG, msg_daemonname,
 			log_buffer);
-	} else if (unlink(path) == -1) {
+	} else if ((unlink(path) == -1) && (errno != ENOENT)) {
```

Comment:
Why we are not considering ENOENT as a failure case ?

Reply:
The vnodes may not be having a topology file associated with it. So we don't want to throw an error when vnodes are getting deleted.